iib/workers/tasks/build_merge_index_image.py

```diff
@@ -249,6 +281,11 @@ def handle_merge_request(
         )
 
         if deprecation_bundles:
+            if is_image_fbc(intermediate_image_name):
+                err_msg = 'File-Based catalog image type is not supported for deprecation yet.'
+                log.error(err_msg)
+                raise IIBError(err_msg)
+
             deprecate_bundles(
```

Comment:
Deprecate_bundles is only run with opm index commands, we don't need to run opm registry deprecatetruncate?

Reply:
we need but when I was creating Jiras I thought that it would be easier to make deprecation separately. 
So that is why we have the code for deprecation FBC here https://github.com/release-engineering/iib/pull/340

Reply:
ah, yes I assumed this PR was based on 340, was directly checking files changed